Week1/github.js

```diff
@@ -0,0 +1,94 @@
+                containerForContributors.innerHTML = "";
+                createAndAppend('p', containerForContributors, { text: 'Contributions' });
+                const contributorsTable = createAndAppend('table', containerForContributors, { id: 'contributorsTable' });
+
+                const contributorsTbody = createAndAppend('tbody', contributorsTable, { id: 'contributorsTbody' });
+               
+                contributors.forEach(contributor => {
+
+                    const contLink = contributor.html_url;
+                    const contImg = contributor.avatar_url;
+                    const contName = contributor.login;
+                    const contNum = contributor.contributions;
+                    const contributorsLink = createAndAppend('a', contributorsTbody, { href: contLink, target: '_blank', class: 'contributorsLink' });
+
+                const contributorsTr = createAndAppend('tr', contributorsLink, { id: `${contributor['id']}`, class: 'contributorsTr' });
+
+                const tdImg = createAndAppend('td', contributorsTr, { id: `${contName}Td`, class: 'inform-contrib' });
+
+                createAndAppend('img', tdImg, { src: contImg, alt: `picture of ${contName}`, class: 'contributor-images' });
+
+                createAndAppend('td', contributorsTr, { text: contName, class: 'inform-contrib contributor-name' });
+
+                createAndAppend('td', contributorsTr, { text: contNum, class: 'inform-contrib contributor-number' });
+            });
+        }
+        catch(err){
+            createAndAppend('div', containerForContributors, { text: err.message, class: 'alert-error' });
+          }
+    }
+    
+
+    async function main(url) {
+       const root = document.getElementById('root');
+       const divForSelect = createAndAppend('div', root, {id: "container-select"});
+       createAndAppend('h5', divForSelect, {text: "HYF Repositories", class: "heading"});
+       const select = createAndAppend('select', divForSelect, {id: "menu"});
+       const container = createAndAppend('div', root, {id: "first-container"});
+       const containerForContributors = createAndAppend('div', root, {id: "second-container"});
+   // use async await method
+    try {
+        const repositories = await fetchJSON(url);
+
+            repositories.sort((a, b) => a.name.localeCompare(b.name));
+
+            repositories.forEach((repository, index) => {
+            createAndAppend('option', select, { text: repository.name, value: index });
+        });
+        
+        select.addEventListener('change', (event) => {
+            renderContributors(containerForContributors, repositories[event.target.value]);
+            renderRepository(container, repositories[event.target.value]);
+        });
+
+        renderRepository(container, repositories[0]);
+        renderContributors(containerForContributors, repositories[0]);
+    }
+    catch(error){
+        container.innerHTML = error.message;
+        return;
```

Comment:
There is no need for this `return` statement, as it is already the last statement that will be executed before the end of the function body.